### ISSUE TITLE
Checkbox ticking

### ISSUE THREAD

**Issue by rexxmagnus** [icewind1991/files_markdown#80]
I love the fact we can now have checkboxes by putting in [ ] or [x], but is it possible to add the feature so that when the checkbox is ticked by clicking in the preview, it puts the x in the source file?

**Comment by icewind1991** [comment]
This should already work in the latest version

**Comment by rexxmagnus** [comment]
I'm using 2.0.1 (in firefox) so I think I've found a bug. 
If I have checkbox items with a blank line between or starting with - it works fine, but if I have the lines close together (terminated with a \ or two spaces) it doesn't modify the correct checkbox in the source.

**Comment by icewind1991** [comment]
Can you post an example text

**Comment by rexxmagnus** [comment]
Sure thing!
```
This works
- [ ] Checkbox 1  
- [ ] Checkbox 2
- [ ] Checkbox 3

The following click incorrectly and replace the first character of the line preceding the checkboxes with an x

This doesn't when you get to checkbox 5  
[ ] Checkbox 4  
[ ] Checkbox 5  
[ ] Checkbox 6  

This doesn't when you get to checkbox 8  
[ ] Checkbox 7\
[ ] Checkbox 8\
[ ] Checkbox 9  

These appear inline (which is fine, as it's the only way to do so)  
[ ] Checkbox 10
[ ] Checkbox 11
[ ] Checkbox 12
```

**Comment by icewind1991** [comment]
Fixed with 2.0.2

**Comment by rexxmagnus** [comment]
Awesome!
